docs/design/contextual-tabs.md

```diff
@@ -0,0 +1,392 @@
+---
+title: Create custom contextual tabs in Office Add-ins
+description: 'Learn how to add custom contextual tabs to your Office Add-in.'
+ms.date: 11/20/2020
+localization_priority: Normal
+---
+
+# Create custom contextual tabs in Office Add-ins (preview)
+
+A contextual tab is a hidden tab control in the Office ribbon that is displayed in the tab row when an object in the Office document, such as an image or a table has focus; for example, the **Table Design** tab that appears on the Excel ribbon when a table is selected. You can include custom contextual tabs in your Office add-in and specify when they are visible or hidden.
+
+> [!NOTE]
+> This article assumes that you are familiar with the following documentation. Please review it if you haven't worked with Add-in Commands (custom menu items and ribbon buttons) recently.
+>
+> - [Basic concepts for Add-in Commands](add-in-commands.md)
+
+> [!IMPORTANT]
+> Custom contextual tabs are in preview. Please experiment with them in a development or testing environment but don't add them to a production add-in.
+>
+> Custom contextual tabs are currently only supported on Excel and only on these platforms and builds:
+>
+> * Excel on Windows (Microsoft 365 only, not perpetual license): Version ???? (Build ?????.?????) Your Microsoft 365 subscription may need to be on the [Current Channel (Preview)](https://insider.office.com/en-us/join/windows) formerly called "Monthly Channel (Targeted)" or "Insider Slow".
```

Comment:
You can use the following November build as a starting point:
https://mroui.trafficmanager.net/ReleaseDetails/Win32/Devmain/Main/C2R/16.0.13519.20000#

Needs to be greater or equal to bld # 16.0.13519.20000.
Insider Beta channel